.github/workflows/release.yml

```
@@ -0,0 +1,157 @@
+        
+    - name: Verify version matches tag
+      run: |
+        if [[ "${{ github.event_name }}" == "push" ]]; then
+          TAG_VERSION="${GITHUB_REF#refs/tags/v}"
+        else
+          TAG_VERSION="${{ github.event.inputs.version }}"
+          TAG_VERSION="${TAG_VERSION#v}"
+        fi
+        
+        PACKAGE_VERSION=$(python -c "import kicad_lib_manager; print(kicad_lib_manager.__version__)")
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)